test/zmumu_pp5tev/tp_from_aod_Data.py

```diff
@@ -0,0 +1,629 @@
+import FWCore.ParameterSet.Config as cms
+
+import subprocess
+
+process = cms.Process("TagProbe")
+
+process.load('Configuration.StandardSequences.Services_cff')
+process.load('FWCore.MessageService.MessageLogger_cfi')
+process.options   = cms.untracked.PSet( wantSummary = cms.untracked.bool(True) )
+process.MessageLogger.cerr.FwkReport.reportEvery = 1000
+
+process.source = cms.Source("PoolSource", 
+    fileNames = cms.untracked.vstring(),
+)
+process.maxEvents = cms.untracked.PSet( input = cms.untracked.int32(-1) )
+
+process.load('Configuration.StandardSequences.GeometryRecoDB_cff')
+process.load('Configuration.StandardSequences.MagneticField_cff')
+process.load('Configuration.StandardSequences.FrontierConditions_GlobalTag_condDBv2_cff')
+process.load("Configuration.StandardSequences.Reconstruction_cff")
+
+process.GlobalTag.globaltag = cms.string('94X_dataRun2_v11')
+
+process.source.fileNames = [
+      '/store/data/Run2017G/SingleMuon/AOD/17Nov2017-v1/910000/E4B056C8-3031-E811-9D14-001E67E6F490.root',
+      ]
+
+## SELECT WHAT DATASET YOU'RE RUNNING ON
+TRIGGER="SingleMu"
+#TRIGGER="DoubleMu"
+
+## ==== Fast Filters ====
+process.goodVertexFilter = cms.EDFilter("VertexSelector",
+    src = cms.InputTag("offlinePrimaryVertices"),
+    cut = cms.string("!isFake && ndof > 4 && abs(z) <= 25 && position.Rho <= 2"),
+    filter = cms.bool(True),
+)
+process.noScraping = cms.EDFilter("FilterOutScraping",
+    applyfilter = cms.untracked.bool(True),
+    debugOn = cms.untracked.bool(False), ## Or 'True' to get some per-event info
+    numtrack = cms.untracked.uint32(10),
+    thresh = cms.untracked.double(0.25)
+)
+
+process.load("HLTrigger.HLTfilters.triggerResultsFilter_cfi")
+
+
+if TRIGGER == "SingleMu":
+    process.triggerResultsFilter.triggerConditions = cms.vstring( 'HLT_HIL2Mu*', 'HLT_HIL3Mu*', 'HLT_HIMu*'  )
```

Comment:
For Z bosons it is better to use 'HLT_HIL3Mu12*'

Reply:
OK, but we could keep it this way for more flexibility in the fitting step right?

Reply:
ok, that should be fine. Though then it would be easier to simply keep only HIL3Mu* muons (because I am not sure what would be the use of L2 or L1 in this case)